### ISSUE TITLE
Overlapping synonyms and incorrect sourcing: MONDO:0015068 & MONDO:0003646

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#7618]
**Mondo term (ID Label)** 
MONDO:0003646 rectum neuroendocrine neoplasm

**Bug/Typo/Error description**
neuroendocrine tumor of rectum, well differentiated, low or intermediate grade is included as a syn, but this is already in use on  MONDO:0015068 neuroendocrine tumor of rectum, well differentiated, low or intermediate grade
And the synonym(s) overlap between these 2 as well, but I think NCI differentiates between tumor and neoplasm in this case:
neuroendocrine tumor/tumour(s) of rectum

http://purl.obolibrary.org/obo/NCIT_C5698 no longer uses 'tumor' as a syn but is cited as the name source for some of the syns on MONDO:0003646, so the names from NCI may need to be reviewed for these (and possibly more?) 

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Comment by sagehrke** [comment]
Dear @kanems,
Thank you for reaching out to the Mondo Disease Ontology, part of the [Monarch Initiative](https://monarchinitiative.org/)!

We have received your ticket and will review it as soon as possible. 

If you have any additional information or updates relevant to your inquiry, please add them in a comment on this ticket. Your input is valuable in helping us understand and resolve the issue effectively.

Thank you for your patience and for your contributions to Mondo! 

Sincerely,
The Mondo Team

**Comment by kanems** [comment]
Following up on this issue, I am not sure about the OrphaNet x-ref (Orphanet_100081) that is currently mapped to MONDO:0003646 ("rectum neuroendocrine **neoplasm**") 
[Neuroendocrine **tumor** of the rectum](https://www.ebi.ac.uk/ols4/ontologies/ordo/classes/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_100081?lang=en)[ordo](https://www.ebi.ac.uk/ols4/ontologies/ordo/classes/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_100081)
I think perhaps this OrphaNet ID is a better fit on MONDO:0015068 rectal neuroendocrine tumor

**Comment by sabrinatoro** [comment]
I agree with @kanems that Orphanet:100081 would be better mapped to MONDO:0015068 rectal neuroendocrine tumor. I will update this mapping
